app/assets/stylesheets/cart.css.scss

```diff
@@ -67,10 +68,10 @@
 }
 
 #cart-total {
-    left: 350px;
-    text-align: center;
-   border: none;
-  border-radius: 5px;
+  left: 350px;
+  text-align: center;
+  border: solid 1px #696969;
+  border-radius: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `5px` instead of `5px 5px 5px 5px`